app/tests/data_factory.py

```diff
@@ -1109,6 +1115,21 @@ def create_samples(self, samples, item, db_create=False, create_asset_file=False
             samples, item=item, db_create=db_create, create_asset_file=create_asset_file, **kwargs
         )
 
+    def get_extension(self, sample_name, kwargs):
```

Comment:
some docstring

Reply:
Its an internal method which should not be used externally, so i'll rename it to `_get_extension` instead of adding a docstring.